plugins/inputs/prometheus/README.md

```diff
@@ -61,6 +61,22 @@ to filter and some tags
   ssl_key = '/path/to/keyfile'
 ```
 
+```toml
+# Use with [Kubernetes headless services](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services)
+[[inputs.prometheus]]
+  # An array of urls to scrape metrics from.
+  urls = ["https://my-headless-service.my-namespace:8080/metrics"]
+
+  # this will look up all the IP addresses behind the host and
+  # query every one of them, using the same schema, port and path
+  do_dns_lookup = true
```

Comment:
I think we should have a separate list of records for dns lookup instead of a boolean, it could be called `kubernetes_services` since I believe this method is slightly different from standard DNS-SD.  I think it would make sense to have these items automatically have the "host" tag.

This will allow combination setups more easily and also I am trying to use less boolean options since they tend to get confusing if you have too many of them.

Reply:
Done!